LISISO/uninstall.sh

```diff
@@ -11,14 +11,14 @@ if [ ${#lisarr[@]} -gt 0 ]; then
 	echo "Uninstalling the Linux Integration Services for Microsoft Hyper-V..."
  	rpm -e ${lisarr[@]}		
 	exit=$?
-        if [ $exit -eq 0 ]; then
-	      echo "Uninstalled Linux Integration Services for Microsoft Hyper-V. Please reboot your system."
+	rm -f /etc/depmod.d/hyperv.conf.rpmsave
```

Comment:
Yes. Since we update this file at installation. so integrity will change and hence rpm will save the hyperv.conf as hyperv.conf.rpmsave. So need to remove it from uninstall script.